libs/crypto/ed25519_impl.cpp

```diff
@@ -16,12 +16,19 @@
  */
 
 #include <ed25519.h>
```

Comment:
I don't really like that this file is called ed25519_impl. We don't have hash_impl, etc. It feels somewhat inconsistent to me.